routers/web/repo/issue.go

```diff
@@ -174,7 +174,11 @@ func issues(ctx *context.Context, milestoneID, projectID int64, isPullOption uti
 	// 0 means issues with no label
 	// blank means labels will not be filtered for issues
 	selectLabels := ctx.FormString("labels")
-	if len(selectLabels) > 0 {
+	if selectLabels == "" {
+		ctx.Data["AllLabels"] = true
+	} else if selectLabels == "0" {
+		ctx.Data["NoLabel"] = true
```

Comment:
That sounds like a bad idea. `0` sounds like a label someone could use.
Can't we use something that no one ever uses, i.e. in the private use area?

Reply:
Or an empty string or something like that?